tools/turbinia.service

```diff
@@ -0,0 +1,29 @@
+# This file should be in /etc/systemd/system/turbinia.service
+# sudo systemctl enable turbinia
+# sudo systemctl status turbinia
+# sudo systemctl start turbinia
+# sudo systemctl stop turbinia
+# sudo systemctl daemon-reload
+
+[Unit]
+Description=Turbinia Daemon
+Documentation=https://github.com/google/turbinia
+Before=multi-user.target
+After=network.target
+After=remote-fs.target
+
+[Service]
+Type=simple
+PIDFile=/var/run/turbinia-daemon.pid
+User=turbinia
+Group=turbinia
+Restart=always
+RestartSec=10
+ExecStartPre=/bin/sh -c 'sudo systemctl set-environment GOOGLE_APPLICATION_CREDENTIALS=/home/turbinia/turbinia-service-account-creds.json'
```

Comment:
I think we should grab this filename from the config rather than hard-coding it here.

Reply:
Done